jsp-integration-addon/src/main/java/org/vaadin/jsp/VaadinTagHandler.java

```diff
@@ -88,4 +91,8 @@ public void setVaadindir(String vaadindir) {
         this.vaadindir = vaadindir;
     }
 
+    public void setDebug(String debug) {
```

Comment:
Please change the parameter to `boolean` and fix the implementation of the method accordingly.